spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala

```diff
@@ -35,6 +35,10 @@ private[tensor] class DenseTensor[@specialized(Float, Double) T: ClassTag](
   var nDimension: Int)(implicit ev: TensorNumeric[T])
   extends Tensor[T] {
 
+  // pre-load MKL library. If we do not do it here,
```

Comment:
If it's in `object DenseTensor`, `new DenseTensor` will not load library automatically.